### PR TITLE
add mongoid support

### PR DESCRIPTION
**Pull request by infernalmaster**
#9

### CODE REVIEW

**Comment by infernalmaster** [comment]
Maybe mongo db is not installed there

File: lib/carrierwave/base64.rb [hunk 1]
```diff
@@ -4,9 +4,19 @@
 module Carrierwave
   module Base64
     class Railtie < Rails::Railtie
-      ActiveSupport.on_load :active_record do
-        require "carrierwave/base64/orm/activerecord"
+
```
Comment (lebedev-yury) [inline lib/carrierwave/base64.rb]: redundant blank space

File: lib/carrierwave/base64.rb [hunk 2]
```diff
@@ -4,9 +4,19 @@
...
+          require "carrierwave/base64/orm/activerecord"
+        end
+      end
+
+      if defined? Mongoid::Document
+        ActiveSupport.on_load :mongoid do
+          require "carrierwave/base64/orm/mongoid"
+        end
       end
+
```
Comment (lebedev-yury) [inline lib/carrierwave/base64.rb]: redundant blank line

File: lib/carrierwave/base64.rb [hunk 3]
```diff
@@ -4,9 +4,19 @@
 module Carrierwave
   module Base64
     class Railtie < Rails::Railtie
-      ActiveSupport.on_load :active_record do
-        require "carrierwave/base64/orm/activerecord"
+
+      if defined? ActiveRecord::Base
```
Comment (lebedev-yury) [inline lib/carrierwave/base64.rb]: this is already checked by `on_load`

File: lib/carrierwave/base64.rb [hunk 4]
```diff
@@ -4,9 +4,19 @@
...
+        ActiveSupport.on_load :active_record do
+          require "carrierwave/base64/orm/activerecord"
+        end
+      end
+
+      if defined? Mongoid::Document
```
Comment (lebedev-yury) [inline lib/carrierwave/base64.rb]: same here

File: lib/carrierwave/base64/orm/mongoid.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+require 'mongoid'
+
+module Carrierwave
+  module Base64
+    module Mongoid
+
+      def mount_base64_uploader(attribute, uploader_class, options = {})
```
Comment (lebedev-yury) [inline lib/carrierwave/base64/orm/mongoid.rb]: this class duplicates the existing adapter for activerecord. We should either make one base class, and inherit from it, or just use one class for both adapters

File: spec/spec_helper.rb [hunk 6]
```diff
@@ -6,8 +6,10 @@
 
 require "rails"
 require "active_record"
+require 'mongoid'
```
Comment (lebedev-yury) [inline spec/spec_helper.rb]: there are doble quotes used everywhere in this file

**Comment by lebedev-yury** [comment]
Mondo is installed on travis, but you should explicitly enable it:

http://docs.travis-ci.com/user/database-setup/#MongoDB

**Comment by aligit** [comment]
Great! Thanks @infernalmaster @lebedev-yury. Do you plan to merge this pull request any time soon?

**Comment by lebedev-yury** [comment]
@aligit: Yes, it will be merged today

**Pull request merged**